src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -354,15 +357,15 @@ namespace HVACUnitarySystem {
 		if ( SELECT_CASE_var == SetPointBased ) {
 			CompOn = 1;
 			if ( present( ZoneEquipment ) ) {
-				ControlUnitarySystemtoSP( UnitarySysNum, FirstHVACIteration, 0, OAUCoilOutTemp, HXUnitOn );
+				ControlUnitarySystemtoSP( UnitarySysNum, 0, FirstHVACIteration, OAUCoilOutTemp, HXUnitOn );
```

Comment:
Rearranged arguments in this function so that all functions had AirLoopNum as 2nd argument.